packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -101,6 +117,16 @@ async function getSignedUrl(fileName: string) {
   return coreGetSignedUrl({ fileName, bucketName, awsRegion: region });
 }
 
+async function isBriefEnabled(aiBrief: string | undefined, cxId: string): Promise<boolean> {
+  const isMrBriefFeatureFlagEnabled = await isMrBriefFeatureFlagEnabledForCx(cxId);
+  return aiBrief === "true" && isMrBriefFeatureFlagEnabled;
```

Comment:
We should prob check if false and return, it avoids executing the promise `isMrBriefFeatureFlagEnabledForCx`

Reply:
good point